README.md

```diff
@@ -20,32 +20,54 @@ This will create the following directory tree for your project:
 
 ## Writting your first project
 
-### Home page
+### Define your api interface
+
+```typescript
+export interface TodoSchema extends Schema {
+  paths: {
+    'POST /todos': {
+      request: {
+        body: {
+          description: string
+        }
+      }
+      response: {
+        200: {
+          content: string;
+        }
+      }
+    }
+  }
+}
+
+const postToExternalApi = async (description: String) : Promise<Task|Error> => {
+  cont result = await tasks.create({ data: { description } })
+  if(result.error){
+    return { ... } : Error
+  }
+  return result.data : Task
+}
+
+const endpoints: Service<TodoSchema> = {
+  'POST /todos': makeService(postToExternalApi)
+};
 ```
+
+### Home page
+
+```typescript
 const Home = ({ todos : Croods<Todo>, ...props }) => {
   return <div/>
 }
 
-export const getServerSideProps = { fetch: useCroodsFetch(options), create: useCroodsCreate(options) }
+export const getServerSideProps = useCroods(endpoints, { name: 'todos' })
```

Comment:
How about this:

```suggestion
export const getServerSideProps = useCroods({ todos: endpoints})
```

So we can have multiple resources in a page

Reply:
The idea would be to have one value containing the whole api.
endpoints could actually be already applied, so when the dev is in this part of the code they only need to call
```typescript
useCroods({ name: 'todos' })
```
Perhaps a better way to achieve multiple resource would be something like:
```typescript
['todos'].map(n => useCroods({ name: n }))
```

Or in a more functional style (assimin R is Ramda):
```typescript
const useCroodsForSeveralResources = R.map(R.compose(useCroods, R.objOf('name')))
```